_pages/about.md

```diff
@@ -21,10 +21,16 @@ You will need to bring a laptop. A tablet will also work, but a laptop is better
 
 ### How do you make money?
 
-We don't actually. It is all 100% volunteer based. The Bradfield Center are sponosoring the venue and the Mentors/Organizers are doing it because they want to inspire young children to pursure careers in STEM.
+We don't actually. It is all 100% volunteer based. The Bradfield Centre are sponsoring the venue and the Mentors/Organizers are doing it because they want to inspire young children to pursue careers in STEM.
 
 # About The Organizers
 
+
+
+[Jamie](https://github.com/JamieDavidson) is currently a professional Software Engineer from Cambridge. Before working as a software engineer, he was Master of Programming at HackLab UK where he created and ran workshops for programming and game design. When he's not at work, Jamie can be found working on one of his many hobby projects. From large scale applications to small side projects running on the Raspberry Pi, Jamie is constantly looking for new challenges and interesting topics to research. He's currently pursuing an interest in data analysis and pattern recognition, and is working to improve his skills at technical writing.
```

Comment:
Why don't we stick our bios into the Volunteer section under coder dojo foundation, otherwise it's a bit redundant